lib/runtime-core/src/vmcalls.rs

```diff
@@ -1,3 +1,5 @@
+//! Unchecked functions for querying the size of memories and growing them.
```

Comment:
Yes, but I think it makes more sense to focus on the fact that these are "vmcalls", that is, these are the implementatons of functions called directly by our generated code. That includes the code for the `memory.grow` opcode, but could include any VM calls in the future. (While I suppose these can also be called from Rust, I don't know whether they are, and they probably shouldn't be.)